src/components/Footer/Footer.jsx

```diff
@@ -38,7 +38,12 @@ const Footer = () => {
   };
 
   useEffect(() => {
-    if (Cookies.get("token")) setLogged(true);
+    if (
+      Cookies.get("Token") ||
+      Cookies.get("OAuthLoginInitiated") ||
+      Cookies.get("isLoggedIn")
+    )
+      setLogged(true);
   }, []);
```

Comment:
I think there is no need to maintain a state for the logged-in part. Wherever needed we can simply directly check the `Cookies.get("isLoggedIn")` property.

Reply:
But we are conditionally rendering the components in the footer. I think we need it. But if you say so I will remove it. 
Awaiting for your reply

Reply:
The place where we are conditionally rendering it, can't we just use the cookies directly ? 

```jsx

{Cookies.get("isLoggedIn") ? <>show this</> :  <>show this</> }
```

Reply:
Hi @YuvarajSingh-0 , can you please do this change so that we can merge this PR ?

Reply:
Right.... Will complete it by the end of the day

Reply:
@tamalCodes completed everything you told. 
Please Review it.